api/src/main/java/openconsensus/OpenConsensus.java

```diff
@@ -0,0 +1,49 @@
+package openconsensus;
+
+import java.util.concurrent.atomic.AtomicReference;
+import openconsensus.trace.Tracer;
+
+/**
+ * Registry provides a global access to observability objects.
+ * An implementation is responsible for setting the Registry.
+ *
+ * @author Pavol Loffay
```

Comment:
I'm not sure we are doing authorship in this codebase

Reply:
Correct, I think we don't need this one.